lib/src/scss-shared/_nx-colors.scss

```diff
@@ -65,3 +65,9 @@ $nx-selectable-colors: (
   yellow: ($nx-yellow-700, $nx-yellow-50),
   lime: ($nx-green-200, $nx-green-50)
 );
+
+// Vulnerability Colours
+$nx-vulnerability-indicator-critical: $nx-red-800;
+$nx-vulnerability-indicator-severe: $nx-orange-700;
+$nx-vulnerability-indicator-moderate: $nx-yellow-700;
+$nx-vulnerability-indicator-low: $nx-blue-600;
```

Comment:
These duplicate the `$nx-threat-color` values above

Reply:
> These duplicate the `$nx-threat-color` values above

You're right. De-duped here: https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/08e5952fea5f1e1540cb124cbcbe35cb322dc15f